packages/x-date-pickers/src/DesktopDatePicker/DesktopDatePicker.test.tsx

```diff
@@ -774,14 +774,6 @@ describe('<DesktopDatePicker />', () => {
     });
   });
 
-  it('should throw console warning when invalid `openTo` prop is provided', () => {
```

Comment:
That one is intersting?

The `warnOnlyOnce` variables are not resetting between tests.
So the 2nd test failed.

I removed it since it's a legacy pickers.
But I don't know how we could cleanly test those warnings.
Maybe just export the variables from the `internals` and reset them to `false` after the test.

Reply:
We could remove the `warnOnce` logic in test env

```js
let warnedOnce = false;
if (!warnedOnce || process.env.NODE_ENV === 'test') {
  console.warn(...)
}
```

Reply:
I'm afraid it would be logged several times and would make the test crash even with `toWarnDev` (which would only catch the first one)
To be tested